bin/fuel-core/src/cli/run/relayer.rs

```diff
@@ -53,10 +50,6 @@ pub struct RelayerArgs {
     pub syncing_log_frequency_secs: u64,
 }
 
-pub fn parse_h160(input: &str) -> Result<H160, <H160 as FromStr>::Err> {
```

Comment:
Would be nice to see a test that verifies the decoded value

Reply:
The function on which you added your comment has been removed. Can you clarify what you want to be tested ?

Reply:
It would be nice to test that parsing of the `H160::from_str` and `Bytes20::from_str` returns the same values=)

Reply:
Done :)